packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -88,15 +89,18 @@ type StyledComponentInputProps = Omit<
   autoCompleteType?: typeof autoCompleteSuggestionTypeAndroid[keyof typeof autoCompleteSuggestionTypeAndroid];
   editable?: boolean;
   onPress?: (event: GestureResponderEvent) => void;
+  $size: NonNullable<BaseInputProps['size']>;
```

Comment:
what does this $ do?

Reply:
> If you want to prevent props meant to be consumed by styled components from being passed to the underlying React node or rendered to the DOM element, you can prefix the prop name with a dollar sign ($), turning it into a transient prop

https://styled-components.com/docs/api#transient-props